DuckDuckGo/NavigationBar/View/AddressBarTextField.swift

```diff
@@ -1054,14 +1058,11 @@ extension AddressBarTextField: NSTextFieldDelegate {
                 suggestionContainerViewModel?.selectPreviousIfPossible()
                 return true
 
-            case #selector(NSResponder.deleteBackward(_:)),
-                 #selector(NSResponder.deleteForward(_:)),
+            case #selector(NSResponder.deleteForward(_:)),
                  #selector(NSResponder.deleteToMark(_:)),
                  #selector(NSResponder.deleteWordForward(_:)),
-                 #selector(NSResponder.deleteWordBackward(_:)),
                  #selector(NSResponder.deleteToEndOfLine(_:)),
                  #selector(NSResponder.deleteToEndOfParagraph(_:)),
-                 #selector(NSResponder.deleteToBeginningOfLine(_:)),
```

Comment:
this breaks the logics of clearing suggestion container view model selected text

Reply:
Can you explain what is being broken in a way I can reproduce?